perf/istio-install/create_cluster.sh

```diff
@@ -165,29 +172,62 @@ export CLUSTER_NAME
 mkdir -p "${WD}/tmp/${CLUSTER_NAME}"
 "${WD}/create_sa.sh" "${GCP_SA}" "${GCP_CTL_SA}"
 
-# shellcheck disable=SC2086
-# shellcheck disable=SC2046
-if [[ "$(gcloud beta container --project "${PROJECT_ID}" clusters list --filter=name="${CLUSTER_NAME}" --format='csv[no-heading](name)')" ]]; then
-  echo "Cluster with this name already created, skipping creation and rerunning init"
-else
-  gc beta container \
+# set config use kpt then create cluster with anthoscli
+function install_with_anthoscli() {
```

Comment:
https://www.gnu.org/software/bash/manual/html_node/Shell-Functions.html

you can just use 
```
install_with_anthoscli() {
...
}
```

if you with to retain the use of function, you might try

```
function install_with_anthoscli {
...
}
```

Reply:
done